### PR TITLE
test: move periph interface lib to pythonlibs path

### PR DESCRIPTION
**Pull request by smlng**
### Contribution description

This PR moves the interface library for periph tests to the
common dist/pythonlibs directory.

### Issues/PRs references

#9409,

### CODE REVIEW

File: tests/periph_i2c/tests/README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ Connect the DUT (device under test) I2C0 to the BPT (bluepill tester).  Informat
 (DUT SCL/D15 <=> BPT SCL1/PB6)</br>
 (DUT SDA/D14 <=> BPT SDA1/PB7)</br>
 (DUT NRST <=> BPT PB1)</br>
-4. Run the test.py from the periph_i2c/tests directory (with various options)</br>
+4. `test.py` requires the path to `$RIOTBASE/dist/pythonlibs` to be added to `PYTHONPATH` environment variable. Either modify `PYTHONPATH` permanently in the `.bashrc` or modify it temporary before invocation:</br>
+`PYTHONPATH=$PYTHONPATH:$RIOTBASE/dist/pythonlibs python3 test.py`
+5. Run the test.py from the periph_i2c/tests directory (with various options)</br>
 `python test.py` for basic tests with autoconnect</br>
 `python3 test.py` for python 3</br>
```
[the remark this answers is not in the document]
Comment (yegorich) [reply tests/periph_i2c/tests/README.md]: I'm for Python3 only. periph_uart doesn't mention python2 in the `README.md`.

[the remark this answers is not in the document]
Comment (smlng) [reply tests/periph_i2c/tests/README.md]: adapted to match periph_uart README

**Comment by smlng** [comment]
@MrKevinWeiss addressed comments - I guess ?

**Comment by smlng** [comment]
but I guess this one is actually obsolete with moving the lib_if to a separate repo, right?

**Comment by smlng** [comment]
yes, closing now

**Pull request closed without merging**